azure-pipelines.yml

```diff
@@ -41,9 +54,7 @@ jobs:
     os: linux
     tox: py37-astropydev -- -n=4
 
-- ${{ if eq(variables['Build.Reason'], 'Schedule') }}:
-  - template: azure-templates.yml
+- ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/') }}:
+  - template: publish-pypi.yml
     parameters:
-      name: Linux_37_numpydev
-      os: linux
-      tox: py37-numpydev -- -n=4
+      dependsOn: [macOS_37_offline,Windows_36_offline,Linux_37_online,Linux_36_Conda_offline,Linux_37_astropydev]
```

Comment:
So do we want to depend on the entire azure CI or just the main 3?

Reply:
I would drop astropy dev but probably leave the others. All three OSes, online and offline and conda are all important.